packaging/kryoptic.spec

```diff
@@ -20,7 +20,7 @@
 
 
 Name:           kryoptic
-Version:        1.0.0
+Version:        1.1.0
```

Comment:
We discussed if we should enable pqc features in the fedora build. The annoying thing is the features list is in 6 places in the spec below. Should we make the change also here as this spec file is used in the CI? Or will it explode with older OpenSSL version we have in CentOS Stream 10 now?

Reply:
it may explode, we'll deal with this later.